config/routes.rb

```diff
@@ -146,6 +146,13 @@
 
       scope module: :forum do
         resources :forums do
+          resources :topics do
+            post 'subscribe', on: :member
+            delete 'unsubscribe', on: :member
+            put 'set_lock', on: :member
```

Comment:
:flushed: I should say thanks, otherwise I'll not know those methods' functionalities.